Makefile

```diff
@@ -55,17 +55,20 @@ link-development-env:
 run: build link-development-env
 	pipenv run flask run
 
-run-gunicorn: link-development-env
-	WEB_SERVER=gunicorn pipenv run ./run_app.sh
+run-gunicorn-gevent: link-development-env
+	WEB_SERVER_TYPE=gunicorn WEB_SERVER_WORKER_CLASS=gevent pipenv run ./run_app.sh
+
+run-gunicorn-gthread: link-development-env
+	WEB_SERVER_TYPE=gunicorn WEB_SERVER_WORKER_CLASS=gthread pipenv run ./run_app.sh
 
 run-uwsgi: link-development-env
-	WEB_SERVER=uwsgi pipenv run ./run_app.sh
+	WEB_SERVER_TYPE=uwsgi pipenv run ./run_app.sh
 
 run-uwsgi-threads: link-development-env
-	WEB_SERVER=uwsgi-threads pipenv run ./run_app.sh
+	WEB_SERVER_TYPE=uwsgi-threads pipenv run ./run_app.sh
 
 run-uwsgi-async: link-development-env
-	WEB_SERVER=uwsgi-async pipenv run ./run_app.sh
+	WEB_SERVER_TYPE=uwsgi-async pipenv run ./run_app.sh
```

Comment:
The run commands for gunicorn and uwsgi are inconsistent. We should consider `WEB_SERVER_TYPE` of `gunicorn-threads` and `gunicorn-async`.

Reply:
Fair point however, we would still need `worker_class = os.getenv("WEB_SERVER_WORKER_CLASS")` given we use a py config file and not cmd line args.

Reply:
I was thinking the env var would be set in the run_app script.

Reply:
Update webserver type to be consistent and to use env var in run_app script.